src/views/token-swap/components/quote-selector/quote-selector.styles.js

```diff
@@ -22,9 +22,6 @@ const useQuoteSelectorStyles = createUseStyles((theme) => ({
   toTokensHeader: {
     flex: "1 1 40%",
   },
-  rewardHeader: {
-    flex: "1 1 20%",
```

Comment:
Token swap use different rewards. They are not related to this issue

Reply:
Thx! Reverted!